src/php/Compiler/Domain/Evaluator/Exceptions/CompiledCodeIsMalformedException.php

```diff
@@ -11,6 +11,23 @@ final class CompiledCodeIsMalformedException extends RuntimeException
 {
     public static function fromThrowable(Throwable $e): self
     {
-        return new self($e->getMessage(), 0, $e);
+        $msg = self::normalize($e->getMessage());
+
+        return new self($msg, 0, $e);
+    }
+
+    private static function normalize(string $msg): string
+    {
+        $pattern = '/Too few arguments to function [^,]+, (\d+) passed in [^,]+ on line (\d+) and exactly (\d+) expected/';
+        if (preg_match($pattern, $msg, $matches)) {
+            return sprintf(
+                'Too few arguments to function on line %s, %s passed in and exactly %s expected',
+                $matches[2],
+                $matches[1],
+                $matches[3],
+            );
+        }
+
+        return $msg;
```

Comment:
I think we could normalize more use cases here. I see there is also other potential error msg like `CompiledCodeIsMalformedException: cannot do cons 1` so we could use another regex for this, and others alike. 
What do you think about this posible solution?